ansible/roles_ocp_workloads/ocp4_workload_5gran_deployments_lab/tasks/pre_workload.yml

```diff
@@ -174,6 +181,46 @@
   ansible.builtin.shell:
     cmd: /etc/NetworkManager/dispatcher.d/forcedns
 
+- name: Ensure disconnected registry service is enabled and running
+  ansible.builtin.systemd:
+    state: restarted
+    daemon_reload: true
+    enabled: true
+    name: podman-registry
+
+- name: Ensure disconnected webcache config files exist
+  ansible.builtin.get_url:
+    # yamllint disable rule:line-length
+    url: "https://raw.githubusercontent.com/RHsyseng/5g-ran-deployments-on-ocp-lab/main/lab-materials/lab-env-data/webcache/podman-webcache.service"
```

Comment:
Can we get the content in in files/?
Or at least a version URL

Reply:
We plan to do that once we test this on dev, next step will be adding a var to control the content.

Reply:
version URL included